src/rewrite_app_repos.sh

```diff
@@ -37,7 +37,10 @@ build_assets() {
   fi
 
   npm ci --prefix "${reponame}" "${reponame}"
-  NODE_OPTIONS=--max_old_space_size=2048 npm run-script --prefix "${reponame}" build
+  for i in {1..5}; do
+    echo "build attempt $i"
+    NODE_OPTIONS=--max_old_space_size=1024 npm run-script --prefix "${reponame}" build && break
```

Comment:
Do we need to add something for when the 5th attempt fails?

Reply:
In that case the for loop will end and the subprocess will exit with error code `1` like before, failing the CI job.